### PR TITLE
Refactoring install ansible in bastion

### PR DESCRIPTION
**Pull request by tei-k**
# Description

Daily terratest failed in AWS.
https://scalar-labs.atlassian.net/browse/DLT-7223

📝 Workaround for Successful Testing.

# Done
- Refactoring `ansible` role.
- Fix resource type of bastion for terratest.
- Upgrade terraform version to 0.12.29 for terratest.

### CODE REVIEW

File: test/modules/awsdeploy/network/terraform.tfvars [hunk 1]
```diff
@@ -16,5 +16,6 @@ additional_public_keys_path = "./additional_public_keys"
 internal_domain = "internal.scalar-labs.com"
 
 network = {
-  bastion_resource_count = 1
+  bastion_resource_count = "1"
+  bastion_resource_type  = "t3.large"
```
Comment (feeblefakie) [inline test/modules/awsdeploy/network/terraform.tfvars:20]: Just a question. Why is it added?

**Review by feeblefakie — APPROVED** [review]
LGTM!

File: test/modules/awsdeploy/network/terraform.tfvars (hunk 1 above)
Comment (tei-k) [reply test/modules/awsdeploy/network/terraform.tfvars:20]: The default `t3.micro` seemed to be failing due to lack of network performance.

Maybe `t3.small` or `t3.medium` will work too.

**Review by ymorimo — APPROVED** [review]
LGTM!

**Pull request merged**